pkgs/development/python-modules/trezor_agent/default.nix

```
@@ -19,13 +21,13 @@ buildPythonPackage rec{
     sha256 = "e82bf000c1178b1a7612f2a90487eb34c6234d2edb15dc8e310ad875d8298690";
   };
 
-  propagatedBuildInputs = [ trezor libagent ecdsa ed25519 mnemonic keepkey semver ];
+  propagatedBuildInputs = [ trezor libagent ecdsa ed25519 mnemonic keepkey semver wheel pinentry ];
```

generally wheel isn't needed after installation.
```suggestion
  nativeBuildInputs = [ wheel ];
  propagatedBuildInputs = [ trezor libagent ecdsa ed25519 mnemonic keepkey semver pinentry ];
```

---

For some reason, this one fails at run time.

---

interesting to need to use it at runtime.... but if that's the case, then disregard :)

---

Indeed, pretty odd.

/cc @romanz as upstream.